datasets/orange_sum/README.md

```diff
@@ -0,0 +1,171 @@
+---
+{
+  "task_categories": [
+    "conditional-text-generation"
+  ],
+  "task_ids": [
+    "summarization"
+  ],
+  "multilinguality": [
+    "monolingual"
+  ],
+  "languages": [
+    "fr"
+  ],
+  "language_creators": [
+    "found"
+  ],
+  "annotations_creators": [
+    "found"
+  ],
+  "source_datasets": [
+    "original"
+  ],
+  "size_categories": [
+    "10K<n<100K"
+  ],
+  "licenses": [
+    "unknown"
+  ]
+}
```

Comment:
We actually expect yaml here, sorry if it was clear (even though the tagger allows to show both json and yaml)


```suggestion
annotations_creators:
- found
language_creators:
- found
languages:
- fr
licenses:
- unknown
multilinguality:
- monolingual
size_categories:
- 10K<n<100K
source_datasets:
- original
task_categories:
- conditional-text-generation
task_ids:
- summarization
```